LICENSE

```diff
@@ -215,3 +215,30 @@ limitations under the License.
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+
+
+---
+
+Portions of this codebase are derived from getsentry/sentry-javascript by Software, Inc. dba Sentry
```

Comment:
Is it *only* the files that have a header comment (like the ones in this PR). I'm just wondering if it's useful to make it obvious which parts this applies to

Reply:
Yeah, something akin to:

> This license only applies to files where explicitly stated in its header.